internal/nix/input.go

```diff
@@ -20,24 +20,50 @@ import (
 	"go.jetpack.io/devbox/internal/lock"
 )
 
+// Input represents a "package" added to the devbox.json config.
+// The word "input" is used because it will be referenced in a generated flake.nix.
```

Comment:
We should rename this struct to package so not sure if this comment will be accurate

Reply:
Whoever does future renames should come here to update the comment then?